src/actions/plugins/easymotion/easymotion.cmd.ts

```diff
@@ -299,8 +299,14 @@ export class EasyMotionWordMoveCommandBase extends BaseEasyMotionCommand {
     vimState: VimState,
     options?: EasyMotion.SearchOptions
   ): EasyMotion.Match[] {
-    // Search for the beginning of all words after the cursor
-    return vimState.easyMotion.sortedSearch(position, new RegExp('\\w{1,}', 'g'), options);
+    const regex = this._options.jumptoanywhere
+      ? configuration.easymotionJumptoanywhereRegex
+        ? new RegExp(configuration.easymotionJumptoanywhereRegex, 'g')
+        : // Default behavior matches beginning & ending of word, camelCase, after _ and after #
+          new RegExp('\b[A-Za-z0-9]|[A-Za-z0-9]\b|_.|#.|[a-z][A-Z]', 'g')
```

Comment:
Is this match intended:

For `MaybeThisWillDo`, the above regex will match the M, e, T, s, W, D

Reply:
This is a better visualization of what I mean: https://regexr.com/3mo29

Reply:
Ah, I see your comment about camelCase, this is probably expected behaviour, but just want to confirm.

Reply:
Hey @jpoon - Yeah, this is definitely intended behavior to match CamelCase. Since the VIM version of `jumptoanywhere` always places the cursor at the end of the match, this behavior pairs perfectly with the `labelPosition: after` setting that was already present in the easymotion API 😎

Here's a sample of what that particular string looks like in action with the jumptoanywhere motion & current default regex:
![2018-03-23 11 43 46](https://user-images.githubusercontent.com/4033621/37848057-066fc3c8-2e90-11e8-82e8-14d2a4fbd8a5.gif)